packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -247,6 +247,30 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     return TestAsyncUtils.guard<void>(() => binding.pump(duration, phase));
   }
 
+  /// Triggers a frame after `duration` amount of time, return as soon as the frame is drawn.
```

Comment:
(maybe clarify that it doesn't actually wait `duration` time, it just fakes it?)

Reply:
Done.